docroot/themes/humsci/humsci_basic/src/scss/components/_vertical-link-card.scss

```diff
@@ -0,0 +1,112 @@
+.vertical-link-card {
+  position: relative;
+  // The margin-top matches the margin-top on both the title and image to prevent
+  // overlap issues.
+  margin-top: hb-calculate-rems(24px);
+  // The margin-left matches the negative margin-left on the title to prevent
+  // overlap issues.
+  margin-left: hb-calculate-rems(31px);
+  padding-bottom: hb-calculate-rems(28px);
+
+  @include hb-colorful {
+    color: $hb-color--white;
+
+    // The gradient adds a 20px hight white block at the top of the linked card
+    // so that the title looks like it has a negative margin on the top when
+    // there isn't an image.
+    background-image: linear-gradient(to bottom, $hb-color--white, $hb-color--white hb-calculate-rems(20px), darken(hb-colorful-variation(tertiary), 30%) hb-calculate-rems(20px));
```

Comment:
can you try spacing this more like so (might also help with it looking intimidating:

```
linear-gradient(
   to bottom,
  $hb-color--white,
  $hb-color--white hb-calculate-rems(20px),
  darken(hb-colorful-variation(tertiary),
  30%) hb-calculate-rems(20px)
);
```

Reply:
I'm pretty sure our scss linter is fine with breaking the gradient onto multiple lines but the CodeClimate linter will errorr.

Reply:
no problemo